### ISSUE TITLE
Wikipedia field not defaulting to local language anymore

### ISSUE THREAD

**Issue by tbertels** [openstreetmap/iD#7890]
It should default to French in this case.

![Presse-papiers-2](https://user-images.githubusercontent.com/3265870/89729190-83f0b900-da33-11ea-88fb-59f540a75852.png)

I think it still worked in April.

Here's a screenshot of the debugger:
![Presse-papiers-1](https://user-images.githubusercontent.com/3265870/89730127-43e20400-da3c-11ea-9031-3407b44ac95b.png)

https://github.com/openstreetmap/iD/blob/874c412b74d9fc16f67abba4d414373fd381643d/modules/ui/fields/wikipedia.js#L146

**Comment by DujaOSM** [comment]
In addition, selecting a Wikidata item automatically sets the Wikipedia link to English one, overriding any previously made input by the user.

The only "correct" editing sequence is to select the Wikipedia language, then pick up the correct article from the dropdown (which is not always obvious because of ambiguity), after which the Wikidata item gets populated automatically. If your target is not en.wp, any other order will likely fight back.

**Comment by tbertels** [comment]
As it currently is, the best workaround seems to copy the full url of the article (with http) and paste it in the Wikipedia field without changing the language, which will then be set automatically. But it requires to go look for the article.

**Comment by boothym** [comment]
I've also come across the same problem as @DujaOSM. I found an object with a non-English wikipedia value but no wikidata, so I clicked in the wikidata field to select the wikidata item - however it changed the wikipedia value to the English one. So then I had to undo and copy & paste the original wikipedia value. 

Basically there should be a way for iD to just fill in the blank wikidata value from wikipedia field.
